assets/kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -250,35 +250,59 @@ spec:
       - name: portieris
         image: {{ .PortierisImage }}
         imagePullPolicy: IfNotPresent
+        command:
+          - "portieris"
+        args:
+          - "--kubeconfig=/etc/openshift/kubeconfig/kubeconfig"
```

Comment:
Support for this flag is currently under review in https://github.com/IBM/portieris/pull/218